### ISSUE TITLE
Vision module for automatic calibration

### ISSUE THREAD

**Issue by h3ndrk** [HULKs/hulk#130]
- Fancy line detection aka feature extraction

Depends on #127

**Comment by tuxbotix** [comment]
Copied from notes at meeting on 2023-05-07 

- Enable fancy line detection via a simple boolean parameter
- Setting stiffness to zero with a simple parameter (which allows to move the head by ourselves)
- Do that in the penalized/initial pose (which is easily entered via the chest button)
- Trigger the new calibration node with its optimization via a simple parameter
- Maybe have a Twix panel for setting these few parameters quickly
- Behavior and other changes in the robotics framework can be postponed
